operations/system_management_health/Troubleshoot_Grafana_Dashboard.md

```diff
@@ -0,0 +1,70 @@
+# Troubleshoot Grafana Dashboard
+
+General Grafana dashboard troubleshooting topics
+  - [Ceph - OSD Overview Dashboard](#ceph-osd-overview-dashboard)
+  - [Ceph - RBD Overview Dashboard](#ceph-rbd-overview-dashboard)
+  - [Ceph - RGW Instance Detail Dashboard](#ceph-rgw-instance-detail-dashboard)
+  - [Ceph - RGW Overview Dashboard](#ceph-rgw-overview-dashboard)
+
+<a name="ceph-osd-overview-dashboard"></a>
+## Ceph - OSD Overview Dashboard: 3 panels not found
+
+Currently skipping cray-sysmgmt-health pie chart plugin installation for airgapped systems. If the system is non airgapped then comment out or remove plugins property in customizations.yaml.
+This will be fixed in a future release.
+
+Example: For airgapped systems (please uncomment plugins property)
+
+```bash
+cray-sysmgmt-health:
+          grafana:
+            externalAuthority: grafana.cmn.{{ network.dns.external }}
+            # Skip plugin installation for airgapped systems.
+            # If the system is non airgapped then you can comment out or remove plugins property.
+            plugins: ""
+```
+
+Example: For non-airgapped systems (please comment out plugins property)
+
+```bash
+cray-sysmgmt-health:
+          grafana:
+            externalAuthority: grafana.cmn.{{ network.dns.external }}
+            # Skip plugin installation for airgapped systems.
+            # If the system is non airgapped then you can comment out or remove plugins property.
+            # plugins: ""
+```
+
+Commands to extract customizations.yaml from the site-init secret.
+
+```bash
+ncn-m001# kubectl -n loftsman get secret site-init -o jsonpath='{.data.customizations\.yaml}' | base64 -d - > customizations.yaml
```

Comment:
Perhaps we should move this command above where we ask them to edit the file?

Reply:
Done